### PR TITLE
Swift3-ready-readme

### PR DESCRIPTION
**Pull request by Jacalz**
This makes the readme reflect update to swift 3.1

### CODE REVIEW

**Comment by Jacalz** [comment]
PR ready for review/merge 😉👍

File: README.md [hunk 1]
```diff
@@ -8,13 +8,9 @@ Brave is based on Firefox iOS, most of the Brave-specific code is in the [brave
...
 Either ensure your app supports [opening urls in a share menu](https://github.com/brave/browser-ios/wiki) or use this project https://github.com/brave/ios-open-thirdparty-browser to open links directly.
 
-## Swift 3 Update
-
-We are currently in the process of migrating Brave from Swift 2.3 to Swift 3.1. To inquire or ask any questions please see #799. All related PRs should be pointed at the associated Swift3 branch.
-
 ## Setup
 
-The master branch (i.e. the mainline) is Xcode8 + Swift2.3
+The master branch (i.e. the mainline) is Xcode8 + Swift3.1
```
Comment (jhreis) [inline README.md]: I think we can just remove this entire line. It is basically saying "our master branch is up-to-date" which should hopefully always be true moving forward.

Comment (Jacalz) [inline README.md]: Fixed

Comment (Jacalz) [inline README.md]: Let's hope so 👍

**Pull request merged**